### PR TITLE
Add Image session reload required handler (#19)

### PR DESCRIPTION
**Pull request by PaGrom**
On Windows 10 EnableFeatureByPackageName throws NullReferenceException because NativeMethod.DismEnableFeature returns exit code 1. I couldn't find docs, but found another dismapi wrapper with using this exit code: https://github.com/jcotton42/ManagedDismapi/blob/master/ManagedDismapi/Interop/NativeMethods.cs#L21
NullReferenceException is thrown because of `Marshal.GetExceptionForHR(errorCode);` returns it for errorcode = 1.
Just added new exception.

### CODE REVIEW

File: src/Microsoft.Dism/NativeConstants.cs [hunk 1]
```diff
@@ -166,5 +166,10 @@ public static partial class DismApi
         /// The requested operation is successful. Changes will not be effective until the service is restarted.
         /// </summary>
         internal const int ERROR_SUCCESS_RESTART_REQUIRED = 0x00000BC3;  // 3011
+
+        /// <summary>
+        /// The requested operation is successful. The DISM session needs to be reloaded.
+        /// </summary>
+        internal const int ERROR_SUCCESS_RELOAD_IMAGE_SESSION_REQUIRED = 0x00000001;  // 1
```
Comment (josemesona) [inline src/Microsoft.Dism/NativeConstants.cs]: Can you please rename this to `DISMAPI_S_RELOAD_IMAGE_SESSION_REQUIRED`?

File: src/Microsoft.Dism/DismException.cs [hunk 2]
```diff
@@ -98,4 +101,17 @@ internal DismRebootRequiredException(int errorCode)
...
         }
     }
-}
\ No newline at end of file
+
+    /// <summary>
+    /// The exception that is thrown when the previous operations requires a reload of image session.
+    /// </summary>
+    [Serializable]
+    public class DismReloadImageSessionRequiredException : DismException
```
Comment (josemesona) [inline src/Microsoft.Dism/DismException.cs]: If there's a known set of method(s) in `DismApi` that will throw this, please add an `<exception cref="" />` like here: https://github.com/josemesona/ManagedDism/blob/dce273148640892935b52da75618ff291986435d/src/Microsoft.Dism/DismApi.cs#L252

**Comment by PaGrom** [comment]
Done.

**Comment by josemesona** [comment]
I've pushed a new package with this change: https://www.nuget.org/packages/Microsoft.Dism/1.6.242

**Comment by PaGrom** [comment]
Great! Thank you! I think you can close #1 and #19 now.

**Pull request merged**